tool/icon_generator/lib/main.dart

```diff
@@ -0,0 +1,149 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'cupertino.dart' as cupertino;
+import 'material.dart' as material;
+
+const String root =
+    '/Users/devoncarew/projects/intellij/flutter-intellij/resources/flutter';
+
+Future main() async {
+  MyIconApp app = MyIconApp(material.icons, cupertino.icons);
+  runApp(app);
+
+  await pumpEventQueue();
+
+  for (material.IconTuple icon in material.icons) {
+    await findAndSave(icon.smallKey, '$root/material/${icon.name}.png',
+        small: true);
+    await findAndSave(icon.largeKey, '$root/material/${icon.name}@2x.png',
+        small: false);
+  }
+
+  for (cupertino.IconTuple icon in cupertino.icons) {
+    await findAndSave(icon.smallKey, '$root/cupertino/${icon.name}.png',
+        small: true);
+    await findAndSave(icon.largeKey, '$root/cupertino/${icon.name}@2x.png',
+        small: false);
+  }
+}
+
+class MyIconApp extends StatelessWidget {
+  MyIconApp(this.materialIcons, this.cupertinoIcons)
+      : super(key: new UniqueKey());
+
+  final List<material.IconTuple> materialIcons; // 16x16
+  final List<cupertino.IconTuple> cupertinoIcons; // 32x32
+
+  @override
+  Widget build(BuildContext context) {
+    const Color color = const Color(0xFF777777);
```

Comment:
comment that this color is just for debugging and won't impact the icons.

Reply:
This is the color that the icon's image is generated in; we want this color as it does well in both IntelliJ's light theme and in Darkula. I'll comment to that effect.

Reply:
<img width="429" alt="screen shot 2018-11-14 at 1 46 07 pm" src="https://user-images.githubusercontent.com/1269969/48514661-b2684280-e813-11e8-8449-71bedd908de0.png">

Reply:
I wish it was more convenient to tint icons in IntelliJ.